.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+### Description
+[Describe what this change achieves]
+
+### Related Issues
+Resolves #[Issue number to be closed when this PR is merged]
+<!-- List any other related issues here -->
+
+### Check List
+- [ ] New functionality includes testing.
```

Comment:
Can we use the leaner version of this checklist that is here?

https://github.com/opensearch-project/OpenSearch/blob/ac78f0a4dc39f5b4be32adc8bf54999b2b44cd14/.github/pull_request_template.md?plain=1#L16-L18

Reply:
This is a standard template though, do you really want me to go and change it everywhere?